include/kodlab_mjbots_sdk/low_pass_filter.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file low_pass_filter.h
+ * @author Lucien C. Peach (peach@seas.upenn.edu)
+ * @brief A discrete-time low-pass filter class.
+ * @date 9/10/2022
+ * 
+ * @copyright 2022 The Trustees of the University of Pennsylvania. All rights reserved.
+ * 
+ */
+
+#pragma once
+
+#include "kodlab_mjbots_sdk/filter.h"
+#include "Eigen/Geometry"
```

Comment:
**[General]** Include order should follow [the style guide](https://drake.mit.edu/styleguide/cppguide.html#Names_and_Order_of_Includes).
```suggestion
#include <Eigen/Geometry>

#include "kodlab_mjbots_sdk/filter.h"
```